lib/standards/aria-roles.js

```diff
@@ -0,0 +1,221 @@
+// Source: https://www.w3.org/TR/wai-aria-1.1/#roles
+
+/* easiest way to see allowed roles is to filter out the global ones
+   from the list of inherited states and properties. The dpub spec
+   does not have the global list so you'll need to copy over from
+   the wai-aria one:
+
+  const globalAttrs = Array.from(
+    document.querySelectorAll('#global_states li')
+  ).map(li => li.textContent.replace(/\s*\(.*\)/, ''));
+
+  const globalRoleAttrs = Array.from(
+    document.querySelectorAll('.role-inherited li')
+  ).filter(li => globalAttrs.includes(li.textContent.replace(/\s*\(.*\)/, '')))
+
+  globalRoleAttrs.forEach(li => li.style.display = 'none');
```

Comment:
What's the point of this comment?

Reply:
Trying to filter out the list of global attributes in the roles supported and inherited lists is difficult when all you want to see is the non-global ones for this object. Since I've had to look at spec a lot and manually filter them out, I wanted to leave a helpful comment for myself and future collaborators on how to see just the non-global attributes.